python/core/__init__.py

```diff
@@ -139,7 +139,12 @@ def __hash__(self):
     NULL = QPyNullVariant(int)
 
 except ImportError:
-    pass
+    try:
+        # TODO: Fixme, this creates an invalid variant, not a NULL one
```

Comment:
Not really a blocker. The major part of the python code isn't updated yet anyway and this one does not affect PyQt4 installations.